src/LICENSE

```diff
@@ -0,0 +1,15 @@
+// sotn-decomp, re-interpretation of the game Castlevania:Symphony of the Night
+// Copyright (C) 2024  Luciano Ciccariello and all the contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
```

Comment:
The license doesn't match the tag you picked `AGPL-3.0-only`

Reply:
adjusted and copied from https://spdx.org/licenses/AGPL-3.0-only.html